File: Tradingsphere_Backend_.Net/TradingApp/Models/Asset.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace TradingApp.Models
{
    [Table("asset")]
    public class Asset
    {
        public long Id { get; set; }

        [Column("symbol")]
        public string Symbol { get; set; } = string.Empty;

        [Column("name")]
        public string Name { get; set; } = string.Empty;

        [Column("asset_type")]
        public string AssetType { get; set; } = string.Empty;

        [Column("exchange")]
        public string Exchange { get; set; } = string.Empty;
    }
}

File: Tradesphere-Backend-Springboot/tradephere-backend-springboot/src/main/java/com/tradesphere/tradepherebackend/model/Asset.java
package com.tradesphere.tradepherebackend.model;

import jakarta.persistence.*;

@Entity
@Table(name = "asset")
public class Asset {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String symbol;

    private String name;

    @Column(name = "asset_type")
    private String assetType;

    // getters and setters
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getSymbol() {
        return symbol;
    }

    public void setSymbol(String symbol) {
        this.symbol = symbol;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAssetType() {
        return assetType;
    }

    public void setAssetType(String assetType) {
        this.assetType = assetType;
    }
}
